#Necessary changes. If not, makepkg will complain about not found header files. Dunno what's the cause

diff --git a/Makefile b/Makefile
index ba0716d..595f713 100644
--- a/Makefile
+++ b/Makefile
@@ -10,8 +10,8 @@ LOG_DIR = libs/log
 DATA_DIR = libs/data
 
 # Quiet (set to @ for a quite compile)
-Q	?= @
-#Q	?= 
+#Q	?= @
+Q	?= 
 
 # Build Tools
 CROSS	?=
@@ -33,7 +33,7 @@ all: Makefile $(EISPICE)
 
 $(EISPICE):
 	@echo Building Module
-	$(Q)python setup.py build $(PYFLAG)
+	$(Q)python2 setup.py build $(PYFLAG)
 	$(Q)cp build/*/simulator_.so module/
 	$(Q)rm -fdr build
 
@@ -77,17 +77,17 @@ clean:
 
 uninstall:
 	@echo Un-Installing
-	$(Q)python uninstall.py
+	$(Q)python2 uninstall.py
 	
 
 install: uninstall
 	@echo Installing
-	$(Q)python setup.py build $(PYFLAG) install
+	$(Q)python2 setup.py build $(PYFLAG) install
 	$(Q)rm -fdr ./build
 
 windist: $(LIBS)
 	@echo Installing
-	$(Q)python setup.py build $(PYFLAG) bdist_wininst
+	$(Q)python2 setup.py build $(PYFLAG) bdist_wininst
 	$(Q)cp ./dist/* .
 	$(Q)rm -fdr ./build
 	$(Q)rm -fdr ./dist
diff --git a/libs/calculon/Makefile b/libs/calculon/Makefile
index 1772710..73295c9 100644
--- a/libs/calculon/Makefile
+++ b/libs/calculon/Makefile
@@ -4,8 +4,8 @@ INC_PREFIX = ../../include
 DATA_LIB = $(LIB_PREFIX)/libdata.a
 
 # Quiet (set to @ for a quite compile)
-Q	?= @
-#Q	?= 
+#Q	?= @
+Q	?= 
 
 # Build Tools
 CROSS	?=
diff --git a/libs/calculon/calc.c b/libs/calculon/calc.c
index b61a085..2bda6d4 100644
--- a/libs/calculon/calc.c
+++ b/libs/calculon/calc.c
@@ -19,9 +19,11 @@
  *
  */
 
-#include <log.h>
+//#include <log.h>
+#include "../../include/log.h"
 #include <math.h>
-#include <data.h>
+//#include <data.h>
+#include "../../include/data.h"
 
 #include "calc.h"
 #include "tokenizer.h"
diff --git a/libs/calculon/parser.c b/libs/calculon/parser.c
index c0b73e0..c9f848d 100644
--- a/libs/calculon/parser.c
+++ b/libs/calculon/parser.c
@@ -29,7 +29,8 @@
 
 	 
 	#include <math.h>
-	#include <log.h>
+	//#include <log.h>
+	#include "../../include/log.h"
 	#include "tokenizer.h"
 	#include "parser.h"
 	
diff --git a/libs/calculon/parser.lem b/libs/calculon/parser.lem
index 3268a95..13ff368 100644
--- a/libs/calculon/parser.lem
+++ b/libs/calculon/parser.lem
@@ -22,7 +22,8 @@
 
 	 
 	#include <math.h>
-	#include <log.h>
+	/*#include <log.h>*/
+	#include "../../include/log.h"
 	#include "tokenizer.h"
 	#include "parser.h"
 	
diff --git a/libs/calculon/tokenizer.c b/libs/calculon/tokenizer.c
index ff2620d..ed48fdc 100644
--- a/libs/calculon/tokenizer.c
+++ b/libs/calculon/tokenizer.c
@@ -22,8 +22,10 @@
  */
 
 #include <math.h>
-#include <log.h>
-#include <data.h>
+//#include <log.h>
+#include "../../include/log.h"
+//#include <data.h>
+#include "../../include/data.h"
 #include <ctype.h>
 
 #include "tokenizer.h"
diff --git a/libs/calculon/tokenizer.h b/libs/calculon/tokenizer.h
index 5ce7157..5607066 100644
--- a/libs/calculon/tokenizer.h
+++ b/libs/calculon/tokenizer.h
@@ -23,7 +23,8 @@
 #ifndef TOKENIZER_H
 #define TOKENIZER_H
 
-#include <data.h>
+//#include <data.h>
+#include "../../include/data.h"
 
 typedef struct _token token_;
 struct _token {
diff --git a/libs/calculon/tokenizer.re b/libs/calculon/tokenizer.re
index 940b2a3..36394b3 100644
--- a/libs/calculon/tokenizer.re
+++ b/libs/calculon/tokenizer.re
@@ -20,8 +20,10 @@
  */
 
 #include <math.h>
-#include <log.h>
-#include <data.h>
+/*#include <log.h>*/
+#include "../../include/log.h"
+/*#include <data.h>*/
+#include "../../include/data.h"
 #include <ctype.h>
 
 #include "tokenizer.h"
diff --git a/libs/data/Makefile b/libs/data/Makefile
index 0af44ad..ec12f0e 100644
--- a/libs/data/Makefile
+++ b/libs/data/Makefile
@@ -3,8 +3,8 @@ LIB_PREFIX	= ../../libs
 INC_PREFIX = ../../include
 
 # Quiet (set to @ for a quite compile)
-Q	?= @
-#Q	?= 
+#Q	?= @
+Q	?= 
 
 # Build Tools
 CROSS	?=
diff --git a/libs/data/data.c b/libs/data/data.c
index bd634ff..f8002bd 100644
--- a/libs/data/data.c
+++ b/libs/data/data.c
@@ -19,7 +19,8 @@
  *
  */
 
-#include <log.h>
+//#include <log.h>
+#include "../../include/log.h"
 
 #include "data.h"
 
diff --git a/libs/data/dblhash.c b/libs/data/dblhash.c
index b38ec08..cdd6084 100644
--- a/libs/data/dblhash.c
+++ b/libs/data/dblhash.c
@@ -7,7 +7,8 @@
 #include <string.h>
 #include <math.h>
 
-#include <log.h>
+//#include <log.h>
+#include "../../include/log.h"
 
 #include "data.h"
 
diff --git a/libs/data/hash.c b/libs/data/hash.c
index d971dfc..87e7f07 100644
--- a/libs/data/hash.c
+++ b/libs/data/hash.c
@@ -6,7 +6,8 @@
 #include <string.h>
 #include <math.h>
 
-#include <log.h>
+//#include <log.h>
+#include "../../include/log.h"
 
 #include "data.h"
 
diff --git a/libs/data/list.c b/libs/data/list.c
index 6c705a5..a0b6451 100644
--- a/libs/data/list.c
+++ b/libs/data/list.c
@@ -19,7 +19,8 @@
  *
  */
 
-#include <log.h>
+//#include <log.h>
+#include "../../include/log.h"
 
 #include "data.h"
 
diff --git a/libs/log/Makefile b/libs/log/Makefile
index 84607f1..74673c0 100644
--- a/libs/log/Makefile
+++ b/libs/log/Makefile
@@ -2,8 +2,8 @@
 INC_PREFIX = ../../include
 
 # Quiet (set to @ for a quite compile)
-Q	?= 
 #Q	?= 
+Q	?= 
 
 INC = log.h
 
diff --git a/libs/netlib/blas/Makefile b/libs/netlib/blas/Makefile
index b6a2fc7..ef485f0 100644
--- a/libs/netlib/blas/Makefile
+++ b/libs/netlib/blas/Makefile
@@ -3,8 +3,8 @@ LIB_PREFIX	= ../..
 INC_PREFIX = ../../../include
 
 # Quiet (set to @ for a quite compile)
-Q	?= @
-#Q	?= 
+#Q	?= @
+Q	?= 
 
 # Build Tools
 CROSS	?=
diff --git a/libs/netlib/cephes/Makefile b/libs/netlib/cephes/Makefile
index 2ac0fa4..25e3687 100644
--- a/libs/netlib/cephes/Makefile
+++ b/libs/netlib/cephes/Makefile
@@ -3,8 +3,8 @@ LIB_PREFIX	= ../..
 INC_PREFIX = ../../../include
 
 # Quiet (set to @ for a quite compile)
-Q	?= @
-#Q	?= 
+#Q	?= @
+Q	?= 
 
 # Build Tools
 CROSS	?=
diff --git a/libs/netlib/lapack/Makefile b/libs/netlib/lapack/Makefile
index de8c5ec..aa8ffdb 100644
--- a/libs/netlib/lapack/Makefile
+++ b/libs/netlib/lapack/Makefile
@@ -3,8 +3,8 @@ LIB_PREFIX	= ../..
 INC_PREFIX = ../../../include
 
 # Quiet (set to @ for a quite compile)
-Q	?= @
-#Q	?= 
+#Q	?= @
+Q	?= 
 
 # Build Tools
 CROSS	?=
diff --git a/libs/netlib/toms/Makefile b/libs/netlib/toms/Makefile
index 2841e04..0f7198e 100644
--- a/libs/netlib/toms/Makefile
+++ b/libs/netlib/toms/Makefile
@@ -3,8 +3,8 @@ LIB_PREFIX	= ../..
 INC_PREFIX = ../../../include
 
 # Quiet (set to @ for a quite compile)
-Q	?= @
-#Q	?= 
+#Q	?= @
+Q	?= 
 
 # Build Tools
 CROSS	?=
diff --git a/libs/simulator/Makefile b/libs/simulator/Makefile
index 803c43f..195bb7c 100644
--- a/libs/simulator/Makefile
+++ b/libs/simulator/Makefile
@@ -10,8 +10,8 @@ TOMS_LIB = $(LIB_PREFIX)/libtoms.a
 CEPHES_LIB = $(LIB_PREFIX)/libcephes.a
 
 # Quiet (set to @ for a quite compile)
-Q	?= @
-#Q	?= 
+#Q	?= @
+Q	?= 
 
 # Build Tools
 CROSS	?=
diff --git a/libs/simulator/core/control.c b/libs/simulator/core/control.c
index 622102b..aa1f952 100644
--- a/libs/simulator/core/control.c
+++ b/libs/simulator/core/control.c
@@ -20,9 +20,11 @@
  */
 
 #include <math.h>
-#include <log.h>
+//#include <log.h>
+#include "../../../include/log.h"
 
-#include "control.h"
+//include "control.h"
+#include "../include/control.h"
 
 /*===========================================================================
  |                          Constructor / Destructor                         |
diff --git a/libs/simulator/core/device.c b/libs/simulator/core/device.c
index 5730538..a137244 100644
--- a/libs/simulator/core/device.c
+++ b/libs/simulator/core/device.c
@@ -19,10 +19,13 @@
  *
  */
 
-#include <log.h>
-#include <data.h>
+//#include <log.h>
+#include "../../../include/log.h"
+//#include <data.h>
+#include "../../../include/data.h"
 
-#include "device_internal.h"
+//#include "device_internal.h"
+#include "../include/device_internal.h"
 
 /*===========================================================================
  |                               Device Control                              |
diff --git a/libs/simulator/core/history.c b/libs/simulator/core/history.c
index 6758147..3e10b50 100644
--- a/libs/simulator/core/history.c
+++ b/libs/simulator/core/history.c
@@ -19,10 +19,13 @@
  *
  */
 
-#include <log.h>
-#include <data.h>
+//#include <log.h>
+#include "../../../include/log.h"
+//#include <data.h>
+#include "../../../include/data.h"
 
-#include "history.h"
+//#include "history.h"
+#include "../include/history.h"
 
 struct _history {
 	double time;
diff --git a/libs/simulator/core/matrix.c b/libs/simulator/core/matrix.c
index af49b5d..fa23470 100644
--- a/libs/simulator/core/matrix.c
+++ b/libs/simulator/core/matrix.c
@@ -19,11 +19,15 @@
  *
  */
 
-#include <log.h>
-#include <superlu.h>
-
-#include "matrix.h"
-#include "history.h"
+//#include <log.h>
+#include "../../../include/log.h"
+//#include <superlu.h>
+#include "../../superlu/superlu.h"
+
+//#include "matrix.h"
+#include "../include/matrix.h"
+//#include "history.h"
+#include "../include/history.h"
 
 typedef int (*matrixLibraryFunction)(matrix_ *r);
 typedef struct _matrixLibrary matrixLibrary_;
diff --git a/libs/simulator/core/node.c b/libs/simulator/core/node.c
index 7aa5988..e8706db 100644
--- a/libs/simulator/core/node.c
+++ b/libs/simulator/core/node.c
@@ -19,9 +19,12 @@
  *
  */
 
-#include <data.h>
-#include <log.h>
-#include "node.h"
+//#include <log.h>
+#include "../../../include/log.h"
+//#include <data.h>
+#include "../../../include/data.h"
+//#include "node.h"
+#include "../include/node.h"
 
 struct _node {
 	int row;
diff --git a/libs/simulator/core/row.c b/libs/simulator/core/row.c
index 733efc9..e7f7809 100644
--- a/libs/simulator/core/row.c
+++ b/libs/simulator/core/row.c
@@ -20,8 +20,10 @@
  */
 
 #include <math.h>
-#include <log.h>
-#include "row.h"
+//#include <log.h>
+#include "../../../include/log.h"
+//#include "row.h"
+#include "../include/row.h"
 
 struct _row {
 	char *name;
diff --git a/libs/simulator/core/simulator.c b/libs/simulator/core/simulator.c
index b3fc952..4d4c5a5 100644
--- a/libs/simulator/core/simulator.c
+++ b/libs/simulator/core/simulator.c
@@ -19,14 +19,20 @@
  *
  */
 
-#include <log.h>
+//#include <log.h>
+#include "../../../include/log.h"
 #include <ctype.h>
-#include <data.h>
-#include <calc.h>
-
-#include "simulator.h"
-#include "device.h"
-#include "history.h"
+//#include <data.h>
+#include "../../../include/data.h"
+//include <calc.h>
+#include "../../calculon/calc.h"
+
+//#include "simulator.h"
+#include "../include/simulator.h"
+//#include "device.h"
+#include "../include/device.h"
+//#include "history.h"
+#include "../include/history.h"
 
 struct _simulator {
 	list_ *devices;
diff --git a/libs/simulator/devices/callback_i.c b/libs/simulator/devices/callback_i.c
index ea99459..963314b 100644
--- a/libs/simulator/devices/callback_i.c
+++ b/libs/simulator/devices/callback_i.c
@@ -20,12 +20,17 @@
  */
 
 #include <math.h>
-#include <log.h>
-#include <data.h>
+//#include <log.h>
+#include "../../../include/log.h"
+//#include <data.h>
+#include "../../../include/data.h"
 
-#include "checkbreak.h"
-#include "checklinear.h"
-#include "device_internal.h"
+//#include "checkbreak.h"
+#include "../include/checkbreak.h"
+//#include "checklinear.h"
+#include "../include/checklinear.h"
+//#include "device_internal.h"
+#include "../include/device_internal.h"
 
 /* Pin Designations */
 #define K			0
diff --git a/libs/simulator/devices/callback_v.c b/libs/simulator/devices/callback_v.c
index e635613..56d9d84 100644
--- a/libs/simulator/devices/callback_v.c
+++ b/libs/simulator/devices/callback_v.c
@@ -20,12 +20,17 @@
  */
 
 #include <math.h>
-#include <log.h>
-#include <data.h>
+//#include <log.h>
+#include "../../../include/log.h"
+//#include <data.h>
+#include "../../../include/data.h"
 
-#include "checkbreak.h"
-#include "checklinear.h"
-#include "device_internal.h"
+//#include "checkbreak.h"
+#include "../include/checkbreak.h"
+//#include "checklinear.h"
+#include "../include/checklinear.h"
+//#include "device_internal.h"
+#include "../include/device_internal.h"
 
 /* Pin Designations */
 #define K			0
diff --git a/libs/simulator/devices/capacitor.c b/libs/simulator/devices/capacitor.c
index 5dfcbad..9460b71 100644
--- a/libs/simulator/devices/capacitor.c
+++ b/libs/simulator/devices/capacitor.c
@@ -20,10 +20,13 @@
  */
 
 #include <math.h>
-#include <log.h>
+//#include <log.h>
+#include "../../../include/log.h"
 
-#include "integrator.h"
-#include "device_internal.h"
+//#include "integrator.h"
+#include "../include/integrator.h"
+//#include "device_internal.h"
+#include "../include/device_internal.h"
 
 /* Pin Designations */
 #define K			0
diff --git a/libs/simulator/devices/inductor.c b/libs/simulator/devices/inductor.c
index 7bcfe40..57a7c04 100644
--- a/libs/simulator/devices/inductor.c
+++ b/libs/simulator/devices/inductor.c
@@ -20,10 +20,13 @@
  */
 
 #include <math.h>
-#include <log.h>
+//#include <log.h>
+#include "../../../include/log.h"
 
-#include "integrator.h"
-#include "device_internal.h"
+//#include "integrator.h"
+#include "../include/integrator.h"
+//#include "device_internal.h"
+#include "../include/device_internal.h"
 
 /* Pin Designations */
 #define K			0
diff --git a/libs/simulator/devices/nonlinear_c.c b/libs/simulator/devices/nonlinear_c.c
index 2679134..81f67f0 100644
--- a/libs/simulator/devices/nonlinear_c.c
+++ b/libs/simulator/devices/nonlinear_c.c
@@ -20,13 +20,19 @@
  */
 
 #include <math.h>
-#include <calc.h>
-#include <data.h>
-#include <log.h>
-
-#include "integrator.h"
-#include "checklinear.h"
-#include "device_internal.h"
+//#include <calc.h>
+#include "../../../include/calc.h"
+//#include <log.h>
+#include "../../../include/log.h"
+//#include <data.h>
+#include "../../../include/data.h"
+
+//#include "integrator.h"
+#include "../include/integrator.h"
+//#include "checklinear.h"
+#include "../include/checklinear.h"
+//#include "device_internal.h"
+#include "../include/device_internal.h"
 
 /* Pin Designations */
 #define K			0
diff --git a/libs/simulator/devices/nonlinear_i.c b/libs/simulator/devices/nonlinear_i.c
index 3da32b1..5795d68 100644
--- a/libs/simulator/devices/nonlinear_i.c
+++ b/libs/simulator/devices/nonlinear_i.c
@@ -20,13 +20,19 @@
  */
 
 #include <math.h>
-#include <calc.h>
-#include <log.h>
-#include <data.h>
-
-#include "checkbreak.h"
-#include "checklinear.h"
-#include "device_internal.h"
+//#include <calc.h>
+#include "../../../include/calc.h"
+//#include <log.h>
+#include "../../../include/log.h"
+//#include <data.h>
+#include "../../../include/data.h"
+
+//#include "checkbreak.h"
+#include "../include/checkbreak.h"
+//#include "checklinear.h"
+#include "../include/checklinear.h"
+//#include "device_internal.h"
+#include "../include/device_internal.h"
 
 /* Pin Designations */
 #define K			0
diff --git a/libs/simulator/devices/nonlinear_v.c b/libs/simulator/devices/nonlinear_v.c
index 12ac957..4013537 100644
--- a/libs/simulator/devices/nonlinear_v.c
+++ b/libs/simulator/devices/nonlinear_v.c
@@ -20,13 +20,19 @@
  */
 
 #include <math.h>
-#include <calc.h>
-#include <log.h>
-#include <data.h>
+//#include <calc.h>
+#include "../../../include/calc.h"
+//#include <log.h>
+#include "../../../include/log.h"
+//#include <data.h>
+#include "../../../include/data.h"
 
-#include "checkbreak.h"
-#include "checklinear.h"
-#include "device_internal.h"
+//#include "checkbreak.h"
+#include "../include/checkbreak.h"
+//#include "checklinear.h"
+#include "../include/checklinear.h"
+//#include "device_internal.h"
+#include "../include/device_internal.h"
 
 /* Pin Designations */
 #define K			0
diff --git a/libs/simulator/devices/resistor.c b/libs/simulator/devices/resistor.c
index a280f60..a279f0d 100644
--- a/libs/simulator/devices/resistor.c
+++ b/libs/simulator/devices/resistor.c
@@ -19,9 +19,11 @@
  *
  */
 
-#include <log.h>
+//#include <log.h>
+#include "../../../include/log.h"
 
-#include "device_internal.h"
+//#include "device_internal.h"
+#include "../include/device_internal.h"
 
 /* Pin Designations */
 #define K			0
diff --git a/libs/simulator/devices/source_i.c b/libs/simulator/devices/source_i.c
index 5af1972..f70abb2 100644
--- a/libs/simulator/devices/source_i.c
+++ b/libs/simulator/devices/source_i.c
@@ -20,11 +20,15 @@
  */
 
 #include <math.h>
-#include <log.h>
-
-#include "checkbreak.h"
-#include "waveform.h"
-#include "device_internal.h"
+//#include <log.h>
+#include "../../../include/log.h"
+
+//#include "checkbreak.h"
+#include "../include/checkbreak.h"
+//#include "waveform.h"
+#include "../include/waveform.h"
+//#include "device_internal.h"
+#include "../include/device_internal.h"
 
 /* Pin Designations */
 #define K			0
diff --git a/libs/simulator/devices/source_v.c b/libs/simulator/devices/source_v.c
index dbe8205..dd81e1b 100644
--- a/libs/simulator/devices/source_v.c
+++ b/libs/simulator/devices/source_v.c
@@ -20,11 +20,15 @@
  */
 
 #include <math.h>
-#include <log.h>
+//#include <log.h>
+#include "../../../include/log.h"
 
-#include "checkbreak.h"
-#include "waveform.h"
-#include "device_internal.h"
+//#include "checkbreak.h"
+#include "../include/checkbreak.h"
+//#include "waveform.h"
+#include "../include/waveform.h"
+//#include "device_internal.h"
+#include "../include/device_internal.h"
 
 /* Pin Designations */
 #define K			0
diff --git a/libs/simulator/devices/tline.c b/libs/simulator/devices/tline.c
index 6b0a6a4..10d4a3a 100644
--- a/libs/simulator/devices/tline.c
+++ b/libs/simulator/devices/tline.c
@@ -20,13 +20,19 @@
  */
 
 #include <math.h>
-#include <log.h>
-#include <data.h>
+//#include <log.h>
+#include "../../../include/log.h"
+//#include <data.h>
+#include "../../../include/data.h"
 
-#include "checkbreak.h"
-#include "history.h"
-#include "history_interp.h"
-#include "device_internal.h"
+//#include "checkbreak.h"
+#include "../include/checkbreak.h"
+//#include "history.h"
+#include "../include/history.h"
+//#include "history_interp.h"
+#include "../include/history_interp.h"
+//#include "device_internal.h"
+#include "../include/device_internal.h"
 
 /* Pin Designations */
 #define K			0
diff --git a/libs/simulator/devices/tline_w.c b/libs/simulator/devices/tline_w.c
index 0fda211..cd1a4d6 100644
--- a/libs/simulator/devices/tline_w.c
+++ b/libs/simulator/devices/tline_w.c
@@ -43,13 +43,18 @@
  */
 
 #define ML 4
-#include <log.h>
+//#include <log.h>
+#include "../../../include/log.h"
 #include <math.h>
 
-#include "device_internal.h"
-#include "mfunc.h"
-#include "complex.h"
-#include "netlib.h"
+//#include "device_internal.h"
+#include "../include/device_internal.h"
+//#include "mfunc.h"
+#include "../include/mfunc.h"
+//#include "complex.h"
+#include "../include/complex.h"
+//#include "netlib.h"
+#include "../include/netlib.h"
 
 /*------------------------ Some useful debug macros -------------------------*/
 
diff --git a/libs/simulator/devices/vicurve.c b/libs/simulator/devices/vicurve.c
index a2bcf2a..671d85c 100644
--- a/libs/simulator/devices/vicurve.c
+++ b/libs/simulator/devices/vicurve.c
@@ -20,12 +20,17 @@
  */
 
 #include <math.h>
-#include <log.h>
+//#include <log.h>
+#include "../../../include/log.h"
 
-#include "checkbreak.h"
-#include "piecewise.h"
-#include "checklinear.h"
-#include "device_internal.h"
+//#include "checkbreak.h"
+#include "../include/checkbreak.h"
+//#include "checklinear.h"
+#include "../include/checklinear.h"
+//#include "piecewise.h"
+#include "../include/piecewise.h"
+//#include "device_internal.h"
+#include "../include/device_internal.h"
 
 /* Pin Designations */
 #define K			0
diff --git a/libs/simulator/include/device.h b/libs/simulator/include/device.h
index 9f8bb3f..697e593 100644
--- a/libs/simulator/include/device.h
+++ b/libs/simulator/include/device.h
@@ -22,7 +22,8 @@
 #ifndef DEVICE_H
 #define DEVICE_H
 
-#include <data.h>
+//#include <data.h>
+#include "../../../include/data.h"
 
 #include "matrix.h"
 #include "control.h"
diff --git a/libs/simulator/include/device_internal.h b/libs/simulator/include/device_internal.h
index b313ed8..134193a 100644
--- a/libs/simulator/include/device_internal.h
+++ b/libs/simulator/include/device_internal.h
@@ -22,7 +22,8 @@
 #ifndef DEVICE_INTERNAL_H
 #define DEVICE_INTERNAL_H
 
-#include <data.h>
+//#include <data.h>
+#include "../../../include/data.h"
 
 #include "device.h"
 
diff --git a/libs/simulator/include/history.h b/libs/simulator/include/history.h
index a630250..c85c75a 100644
--- a/libs/simulator/include/history.h
+++ b/libs/simulator/include/history.h
@@ -22,7 +22,8 @@
 #ifndef HISTORY_H
 #define HISTORY_H
 
-#include <data.h>
+//#include <data.h>
+#include "../../../include/data.h"
 
 typedef struct _history history_;
 
diff --git a/libs/simulator/include/matrix.h b/libs/simulator/include/matrix.h
index 5088c60..9053ce1 100644
--- a/libs/simulator/include/matrix.h
+++ b/libs/simulator/include/matrix.h
@@ -22,7 +22,8 @@
 #ifndef MATRIX_H
 #define MATRIX_H
 
-#include <data.h>
+//#include <data.h>
+#include "../../../include/data.h"
 #include "row.h"
 #include "node.h"
 #include "control.h"
diff --git a/libs/simulator/include/row.h b/libs/simulator/include/row.h
index e2ce952..12e5e16 100644
--- a/libs/simulator/include/row.h
+++ b/libs/simulator/include/row.h
@@ -22,7 +22,8 @@
 #ifndef ROW_H
 #define ROW_H
 
-#include <data.h>
+//#include <data.h>
+#include "../../../include/data.h"
 
 typedef struct _row row_;
 
diff --git a/libs/simulator/math/checkbreak.c b/libs/simulator/math/checkbreak.c
index b53b594..bbf3f1b 100644
--- a/libs/simulator/math/checkbreak.c
+++ b/libs/simulator/math/checkbreak.c
@@ -20,10 +20,13 @@
  */
 
 #include <math.h>
-#include <log.h>
+//#include <log.h>
+#include "../../../include/log.h"
 
-#include "control.h"
-#include "checkbreak.h"
+//#include "control.h"
+#include "../include/control.h"
+//#include "checkbreak.h"
+#include "../include/checkbreak.h"
 
 #define N	3
 #define MaxAbs(x,y) ((fabs(x) > fabs(y)) ? fabs(x) : fabs(y))
diff --git a/libs/simulator/math/checklinear.c b/libs/simulator/math/checklinear.c
index 27bdf7f..258d8cf 100644
--- a/libs/simulator/math/checklinear.c
+++ b/libs/simulator/math/checklinear.c
@@ -20,10 +20,13 @@
  */
 
 #include <math.h>
-#include <log.h>
+//#include <log.h>
+#include "../../../include/log.h"
 
-#include "control.h"
-#include "checklinear.h"
+//#include "control.h"
+#include "../include/control.h"
+//#include "checklinear.h"
+#include "../include/checklinear.h"
 
 #define MaxAbs(x,y) ((fabs(x) > fabs(y)) ? fabs(x) : fabs(y))
 
diff --git a/libs/simulator/math/complex.c b/libs/simulator/math/complex.c
index d827a5f..6f70b26 100644
--- a/libs/simulator/math/complex.c
+++ b/libs/simulator/math/complex.c
@@ -21,9 +21,11 @@
 
 #include <math.h>
 #include <float.h>
-#include <log.h>
+//#include <log.h>
+#include "../../../include/log.h"
 
-#include "complex.h"
+//#include "complex.h"
+#include "../include/complex.h"
 
 /*===========================================================================*/
 
diff --git a/libs/simulator/math/history_interp.c b/libs/simulator/math/history_interp.c
index 8555e26..91f5d49 100644
--- a/libs/simulator/math/history_interp.c
+++ b/libs/simulator/math/history_interp.c
@@ -20,11 +20,15 @@
  */
 
 #include <math.h>
-#include <log.h>
-
-#include "row.h"
-#include "history.h"
-#include "history_interp.h"
+//#include <log.h>
+#include "../../../include/log.h"
+
+//#include "row.h"
+#include "../include/row.h"
+//#include "history.h"
+#include "../include/history.h"
+//#include "history_interp.h"
+#include "../include/history_interp.h"
 
 #define MaxAbs(x,y) ((fabs(x) > fabs(y)) ? fabs(x) : fabs(y))
 
diff --git a/libs/simulator/math/integrator.c b/libs/simulator/math/integrator.c
index 3d0a751..d16d428 100644
--- a/libs/simulator/math/integrator.c
+++ b/libs/simulator/math/integrator.c
@@ -20,10 +20,13 @@
  */
 
 #include <math.h>
-#include <log.h>
+//#include <log.h>
+#include "../../../include/log.h"
 
-#include "control.h"
-#include "integrator.h"
+//#include "control.h"
+#include "../include/checklinear.h"
+//#include "integrator.h"
+#include "../include/integrator.h"
 
 #define MaxAbs(x,y) ((fabs(x) > fabs(y)) ? fabs(x) : fabs(y))
 
diff --git a/libs/simulator/math/mfunc.c b/libs/simulator/math/mfunc.c
index 567f8ae..9333e39 100644
--- a/libs/simulator/math/mfunc.c
+++ b/libs/simulator/math/mfunc.c
@@ -22,10 +22,13 @@
 #include <math.h>
 #include <float.h>
 #define ML 4
-#include <log.h>
+//#include <log.h>
+#include "../../../include/log.h"
 
-#include "complex.h"
-#include "netlib.h"
+//#include "complex.h"
+#include "../include/complex.h"
+//#include "netlib.h"
+#include "../include/netlib.h"
 
 /*===========================================================================*/
 
diff --git a/libs/simulator/math/netlib.c b/libs/simulator/math/netlib.c
index 16222c6..6ddef2f 100644
--- a/libs/simulator/math/netlib.c
+++ b/libs/simulator/math/netlib.c
@@ -20,9 +20,11 @@
  */
 
 #define ML 4
-#include <log.h>
+//#include <log.h>
+#include "../../../include/log.h"
 
-#include "complex.h"
+//#include "complex.h"
+#include "../include/complex.h"
 
 /*================= LAPACK / BLAS / TOMS Fortran Functions ==================*/
 
diff --git a/libs/simulator/math/piecewise.c b/libs/simulator/math/piecewise.c
index 773afef..b97c7db 100644
--- a/libs/simulator/math/piecewise.c
+++ b/libs/simulator/math/piecewise.c
@@ -21,9 +21,11 @@
 
 #include <math.h>
 
-#include <log.h>
+//#include <log.h>
+#include "../../../include/log.h"
 
-#include "piecewise.h"
+//#include "piecewise.h"
+#include "../include/piecewise.h"
 
 typedef struct {
 	double x;
diff --git a/libs/simulator/math/waveform.c b/libs/simulator/math/waveform.c
index 6c49672..c505b33 100644
--- a/libs/simulator/math/waveform.c
+++ b/libs/simulator/math/waveform.c
@@ -20,12 +20,17 @@
  */
 
 #include <math.h>
-#include <log.h>
+//#include <log.h>
+#include "../../../include/log.h"
 
-#include "piecewise.h"
-#include "control.h"
-#include "waveform.h"
-#include "netlib.h"
+//#include "piecewise.h"
+#include "../include/piecewise.h"
+//#include "control.h"
+#include "../include/control.h"
+//#include "waveform.h"
+#include "../include/waveform.h"
+//#include "netlib.h"
+#include "../include/netlib.h"
 
 struct _waveform {
 	char type;	/* Type of Source
diff --git a/libs/superlu/Makefile b/libs/superlu/Makefile
index 282a37e..a327b3e 100644
--- a/libs/superlu/Makefile
+++ b/libs/superlu/Makefile
@@ -3,8 +3,8 @@ LIB_PREFIX	= ../../libs
 INC_PREFIX = ../../include
 
 # Quiet (set to @ for a quite compile)
-Q	?= @
-#Q	?= 
+#Q	?= @
+Q	?= 
 
 # Build Tools
 CROSS	?=
diff --git a/uninstall.py b/uninstall.py
index d2b92ff..6d0be3a 100644
--- a/uninstall.py
+++ b/uninstall.py
@@ -1,7 +1,7 @@
 import os
 from distutils.sysconfig import get_python_lib
 
-top = get_python_lib() + '\eispice'
+top = get_python_lib() + '/eispice'
 
 print top
 
--- a/module/simulatormodule.c
+++ b/module/simulatormodule.c
@@ -28,9 +28,11 @@
 #define NPY_OWNDATA OWNDATA
 #endif
 
-#include <log.h>
+//#include <log.h>
+#include "../include/log.h"
 LogMaster;
-#include <simulator.h>
+//#include <simulator.h>
+#include "../libs/simulator/include/simulator.h"
 
 /*===========================================================================
  |                                 Waveforms                                 |


